obspy/conftest.py

```diff
@@ -145,15 +144,8 @@ def pytest_configure(config):
     import matplotlib
     matplotlib.use('Agg')
 
-    # Register markers. We should really do this in pytest.ini or the like
-    # but the existence of that file messed with the config.py hooks for
-    # some reason.
-    config.addinivalue_line(
-        "markers", "network: Test requires network resources (internet)."
-    )
-    config.addinivalue_line(
-        "markers", "image: Test produces a matplotlib image."
-    )
+    # do not collect tests in arclink module
+    pytest.importorskip('obspy.clients.arclink')
```

Comment:
The comment is wrong. It should be `# ignore possible import error due to DepreciationWarning when using -W error flag`. arclink is the only module which raises a Warning upon import. The other modules only warn when a function is called. Adding the other deprecated modules here, would have no effect, because this command only skips the imports when an error is encountered.

Reply:
Lets remember to remove this  in #2994